### PR TITLE
[MISC] Add Sharg 1.0.0 release.

### PR DESCRIPTION
**Pull request by smehringer**

### CODE REVIEW

File: _posts/2022-10-10-sharg-release.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+#### The new [`sharg::config`](https://docs.seqan.de/sharg/1.0.0/structsharg_1_1config.html) design
+
+The new config design is also an API break because we do not support the former calls from the `seqan3::argument_parser`.
+
+We removed the `sharg::option_spec` as it is obsolete in the new API.
+
+#### Validators
+
+To avoid being dependent on the SeqAn3 I/O module, you now have to give a list of file extensions explicitly to `sharg::input_file_validator` and `sharg::output_file_validator`:
+
```
Comment (eseiler) [inline _posts/2022-10-10-sharg-release.md:89]:
```suggestion
{% raw %}
```

File: _posts/2022-10-10-sharg-release.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+```cpp
+sharg::input_file_validator validator{std::vector<std::string>{{"exe"}, {"fasta"}}};`.
```
Comment (eseiler) [inline _posts/2022-10-10-sharg-release.md:91]:
```suggestion
sharg::input_file_validator validator{std::vector<std::string>{{"exe"}, {"fasta"}}};
```

File: _posts/2022-10-10-sharg-release.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+We removed the `sharg::option_spec` as it is obsolete in the new API.
+
+#### Validators
+
+To avoid being dependent on the SeqAn3 I/O module, you now have to give a list of file extensions explicitly to `sharg::input_file_validator` and `sharg::output_file_validator`:
+
+```cpp
+sharg::input_file_validator validator{std::vector<std::string>{{"exe"}, {"fasta"}}};`. 
+```
+Please follow [the SeqAn3 issue](https://github.com/seqan/seqan3/issues/2927) to see how the file extensions can be extracted from SeqAn3 files.
```
Comment (eseiler) [inline _posts/2022-10-10-sharg-release.md:93]:
```suggestion
{% endraw %}
Please follow [the SeqAn3 issue](https://github.com/seqan/seqan3/issues/2927) to see how the file extensions can be extracted from SeqAn3 files.
```

**Comment by eseiler** [comment]
I escaped the syntax that was confusing Liquid (`}}}` or something).

https://www-seqan-de-git-fork-smehringer-master-seqan.vercel.app/news/
The short description seems rather long compared to the other news.

File: _posts/2022-10-10-sharg-release.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+We are thrilled to announce the first **Sharg parser release: 1.0.0** with most of the API being **stable**.
+
+We have outsourced the SeqAn3 Argument Parser to its own light-weight, dependency-free repository: 
+The [**Sharg parser**](https://github.com/seqan/sharg-parser). 
+Most of the **API stayed similar**. We added a **new, flexible config design** for adding options and flags, improving readability 
+and maintainability. Not unexpectedly, we have some **changes in namespace and naming**: 
+
+The former <code>seqan3::argument_parser</code> is now the <code>sharg::parser</code>
+
+<!--more-->
```
Comment (smehringer) [inline _posts/2022-10-10-sharg-release.md:19]:
```suggestion
The [**Sharg parser**](https://github.com/seqan/sharg-parser). 

<!--more-->

Most of the **API stayed similar**. We added a **new, flexible config design** for adding options and flags, improving readability 
and maintainability. Not unexpectedly, we have some **changes in namespace and naming**: 

The former <code>seqan3::argument_parser</code> is now the <code>sharg::parser</code>
```

**Review by eseiler — APPROVED** [review]

**Pull request merged**